AUTHORS

```diff
@@ -1079,6 +1079,7 @@ Miguel Torres Costa <miguelptcosta1995@gmail.com>
 Rikard Nordgren <rikard.nordgren@farmaci.uu.se>
 Arun sanganal <74652697+ArunSanganal@users.noreply.github.com>
 Kamlesh Joshi <72374645+kamleshjoshi8102@users.noreply.github.com>
+oriel <orielmalihi1@gmail.com>
```

Comment:
The `.mailmap` file should be used to specify how you want your name to appear in the AUTHORS file. Here the name "oriel" comes from your git config.

Should it actually be "Oriel Malihi" or something similar?

Reply:
If so then actually the .mailmap should have two lines like:
```
Oriel Malihi <orielmalihi1@gmail.com> oriel <orielmalihi1@gmail.com>
Oriel Malihi <orielmalihi1@gmail.com> orielmalihi <57680696+orielmalihi@users.noreply.github.com>
```
Then if you rerun the `authors_update.py` file it will only add a single author listed as:
```
Oriel Malihi <orielmalihi1@gmail.com>
```

Reply:
@oscarbenjamin now I get different error, 
can you tell why?

Reply:
I added this two lines (instead of the libe you wrote before),
byt now I get different error, 
@oscarbenjamin can you tell why?

Reply:
The failure says that the AUTHORS file was updated.

Checking out the PR I get:
```console
$ bin/mailmap_update.py 
checking git authors...

checking .mailmap...
$ bin/authors_update.py 
No changes made to AUTHORS.
```
That's the expected output.

The problem is that the AUTHORS file has already been updated on the master branch because another PR has been merged by a new contributor. You need to merge in master and rerun `authors_update.py`. Actually using merge doesn't work because GitHub merges the PR in the other direction (PR into master rather than master into the PR) which changes the ordering of the commits.

You need to rebase the PR onto master like this:
```console
$ git rebase upstream/master
$ bin/authors_update.py 
The AUTHORS file was updated.
Changes were made in the authors file
$ git diff
diff --git a/AUTHORS b/AUTHORS
index e86b934b8b..6619a250a4 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -1079,7 +1079,6 @@ Miguel Torres Costa <miguelptcosta1995@gmail.com>
 Rikard Nordgren <rikard.nordgren@farmaci.uu.se>
 Arun sanganal <74652697+ArunSanganal@users.noreply.github.com>
 Kamlesh Joshi <72374645+kamleshjoshi8102@users.noreply.github.com>
-Oriel Malihi <orielmalihi1@gmail.com>
 Joseph Rance <56409230+Joseph-Rance@users.noreply.github.com>
 Huangduirong <huangduirong@huawei.com>
 Nils Schulte <47043622+Schnilz@users.noreply.github.com>
@@ -1098,3 +1097,4 @@ Rahil Parikh <r.parikh@somaiya.edu>
 Jason Ross <jasonross1024@gmail.com>
 Joannah Nanjekye <joannah.nanjekye@ibm.com>
 Ayush Kumar <ayushk7102@gmail.com>
+Oriel Malihi <orielmalihi1@gmail.com>
```

Reply:
ok,
it seems to be wotking for now.
Thanks!